src/components/Profile/List/__test__/ListProfiles.test.jsx

```diff
@@ -0,0 +1,127 @@
+delete window.matchMedia;
+window.matchMedia = (query) => ({
+  matches: false,
+  media: query,
+  onchange: null,
+  addListener: jest.fn(),
+  removeListener: jest.fn(),
+});
+
+import React from "react";
+
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+import { useGetProfileListQuery } from "../../../../api/profileApi";
+import ListProfiles from "..";
+
+import "@testing-library/jest-dom/extend-expect";
```

Comment:
import sequence is not seems correct

Reply:
kk i will check the sequence and make it correct =